driver-core/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -1,7 +1,35 @@
 {
   "resources":{
   "includes":[{
+    "pattern":"\\QMETA-INF/services/com.mongodb.spi.dns.DnsClientProvider\\E"
```

Comment:
Aren't you worried by the cost of these JNI registrations?  It will make the resulting image quite big.

Reply:
Indeed, thank you. We noticed that specifying shared libraries in `resource-config.json` significantly increases the size of the produced native image. I'll remove all such entries from our JARs, and move them back to the `:graalvm-native-image-app` Gradle project. We'll document that users have to specify them in their own configs if needed, and will probably refer them to `graalvm-native-image-app/src/main/resources/META-INF/native-image/resource-config.json` as an example.

Reply:
Yes, we now have those entries only in `graalvm-native-image-app/src/main/resources/META-INF/native-image/resource-config.json`, and will suggest a user to specify them on his own if needed.